recipe/meta.yaml

```diff
@@ -12,21 +12,20 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1000
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
   build:
+    - {{ compiler('c') }}
+  host:
     - python
-    - setuptools
-    - numpy 1.8.*  # [not (win and (py35 or py36))]
-    - numpy 1.9.*  # [win and py35]
-    - numpy 1.11.*  # [win and py36]
```

Comment:
@jakirkham, to me it looks like that numpy is actually not a build-time dependency but merely needed as an import. Does it still have to go into hosts in this case?

Reply:
Oh, I assumed it was since it was listed here. If not, yeah this could be dropped.

FWIW looked through the code as well and saw [this reference during the build]( https://github.com/biopython/biopython/blob/biopython-172/setup.py#L197 ) and several references to NumPy in comments in [this C file]( https://github.com/biopython/biopython/blob/biopython-172/Bio/KDTree/KDTreemodule.c ). That said, it doesn't appear to use NumPy headers anywhere and the C file is using the Python Buffer Protocol, which while it works on NumPy arrays is not NumPy specific and is just part of the Python spec. IOW completely agree it doesn't seem to be needed.

Reply:
Yes, touched on in another comment: Historically Biopython did have C code which compiled against NumPy. Since Biopython 1.72 (current release) this is no longer the case, by using the Python Buffer Protocol the C code only needs NumPy at run time (import time).